graphql2/cmtype.go

```diff
@@ -1,17 +1,21 @@
 package graphql2
 
 import (
-	"github.com/target/goalert/user/contactmethod"
 	"io"
+	"log"
 	"strings"
 
+	"github.com/target/goalert/user/contactmethod"
+
 	graphql "github.com/99designs/gqlgen/graphql"
 	"github.com/pkg/errors"
 )
 
 func MarshalContactMethodType(t contactmethod.Type) graphql.Marshaler {
 	return graphql.WriterFunc(func(w io.Writer) {
-		io.WriteString(w, `"`+string(t)+`"`)
+		if _, err := io.WriteString(w, `"`+string(t)+`"`); err != nil {
+			log.Println("ERROR with MarshalContactMethodType when using io.WriteString:", err)
```

Comment:
Does this writer write to the ResponseWriter or a buffer?

If it's the former, we likely should ignore it because it means the other end is disconnected; if it's later, it will never return error (panic internal). In both cases, it should be ignored.

Lastly:
- don't use the stdlib logger; use the application logging package
- use the proper error message formatting

Reply:
I'll just have it ignore the error since we probably don't care about it.  Let me know if you'd rather a panic instead.

(as an fyi, i didn't use the app logger here because ctx didn't exist, and it would've involved application changes to get ctx passed in here, which I figured was outside the scope of work).  Either way it's moot once I set it to ignore the error.

Reply:
TBH, probably why we did that to begin with. The complexity of getting context here (since `gqlgen` doesn't support it here) would not be worth it.